src/PrestaShopBundle/Translation/Api/StockApi.php

```diff
@@ -70,14 +70,16 @@ public function getTranslations()
             'product_low_stock_alert' => $this->translator->trans('Low stock alert:', [], 'Admin.Catalog.Feature'),
             'title_available' => $this->translator->trans('Available', [], 'Admin.Global'),
             'title_bulk' => $this->translator->trans('Bulk edit quantity', [], 'Admin.Catalog.Feature'),
+            'title_combination_id' => $this->translator->trans('ID Combination', [], 'Admin.Catalog.Feature'),
             'title_date' => $this->translator->trans('Date & Time', [], 'Admin.Catalog.Feature'),
             'title_edit_quantity' => $this->translator->trans('Edit quantity', [], 'Admin.Catalog.Feature'),
             'title_employee' => $this->translator->trans('Employee', [], 'Admin.Global'),
             'title_import' => $this->translator->trans('Go to the import system', [], 'Admin.Catalog.Feature'),
             'title_export' => $this->translator->trans('Export data into CSV', [], 'Admin.Catalog.Feature'),
             'title_movements_type' => $this->translator->trans('Type', [], 'Admin.Global'),
             'title_physical' => $this->translator->trans('Physical', [], 'Admin.Catalog.Feature'),
-            'title_product' => $this->translator->trans('Product', [], 'Admin.Global'),
+            'title_product' => $this->translator->trans('Product (name)', [], 'Admin.Global'),
+            'title_product_id' => $this->translator->trans('ID', [], 'Admin.Catalog.Feature'),
```

Comment:
```suggestion
            'title_product_id' => $this->translator->trans('ID', [], 'Admin.Global'),
```

Reply:
🤔  Maybe here it's better specify another domain? Or there'll be the risk to change the translations also in others page in the BO, wdyt?

Reply:
Yes, we could be more specific here. However, I'm not sure we'll be able to be super specific as this string will probably appear on the product page, which is in Admin.Catalog as well... 😅